theia-full-docker/Dockerfile

```diff
@@ -316,6 +316,9 @@ yarn --cache-folder ./ycache && rm -rf ./ycache && \
      NODE_OPTIONS="--max_old_space_size=4096" yarn theia build && yarn theia download:plugins \
 ;fi
 
+# Change permissions to make the `yang-language-server` executable.
```

Comment:
Paul's fix has been merged. We'll still need this work-around for `latest` until end of month.